src/components/AgreementLinesFieldArray/AgreementLineField.js

```diff
@@ -84,8 +84,7 @@ export default class AgreementLineField extends React.Component {
   }
 
   renderCoverage = (resource) => {
-    const coverage = get(resource, 'coverage');
-    return <CoverageStatement statements={coverage} />;
+    return <SerialCoverage statements={resource.coverage} />;
```

Comment:
How do you know this is a serial? Couldn't the entitlement be for a book and thus a monograph?

Reply:
Speaking to Gill, she mentioned that we wanted the behaviour in the edit screen to remain the same for the time being, so I used `SerialCoverage` for now, but this obviously would be very easy to change in future if needs be.

Reply:
Comment added